src/boilerplate.ts

```diff
@@ -146,6 +146,9 @@ export const install = async (toolbox: IgniteToolbox) => {
   spinner.start()
   const boilerplatePath = `${__dirname}/../boilerplate`
   const copyOpts = { overwrite: true, matching: "!*.ejs" }
+  if (!useExpo) {
+    filesystem.remove(`${boilerplatePath}/app/theme/fonts/index.ts`)
```

Comment:
This looks like it's removing the boilerplate's version of this file, which is kinda dangerous as I could be referring to a local folder. Instead, let's copy it over with the rest of the files, and then delete it from the freshly generated app afterward.